src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -567,6 +665,22 @@ export const GenericTable = ({
     } else deleteRow();
   }, [deleteRow]);
 
+  const revertTableWithDatasetData = useCallback(
+    (isChecked) => {
+      localStorage.setItem('dontAskAgainToRevertTableData', isChecked);
+      closeRevertDialog();
+      _getDataFromTwingraphDataset(parameter.id ? updateParameterValue : updateParameterValueWithReset);
+      if (parameter.id) updateOnFirstEdition();
```

Comment:
This part is not intuitive, maybe we could add a small comment to explain the logic?